web/src/containers/activity/Overview.js

```diff
@@ -0,0 +1,144 @@
+import { connect } from 'react-redux';
+
+import { t } from '../../i18n';
+import { updateActivity as updateActivityAction } from '../../actions/activities';
+import { RichText } from '../../components/Inputs';
+import Instruction from '../../components/Instruction';
+import { Subsection } from '../../components/Section';
+import { selectActivityByKey } from '../../reducers/activities.selectors';
+
+const Description = props => {
+  const { activity, updateActivity } = props;
+  const { alternatives, description, summary } = activity;
+
+  const overviewLabel = useMemo(
+    () =>
+      t('activities.overview.activityOverviewInput.label', {
+        defaultValue: 'Provide a short overview of the activity.'
+      }),
+    []
+  );
+  const overviewHint = useMemo(
+    () =>
+      t('activities.overview.activityOverviewInput.hint', {
+        defaultValue: ''
+      }),
+    []
+  );
+  const overviewOnChange = useCallback(
+    ({ target: { value } }) => updateActivity(activity.key, { summary: value }),
+    [activity.key]
+  );
+
+  const descriptionLabel = useMemo(
+    () =>
+      t(
+        `activities.overview.activityDescriptionInput.${
+          activity.fundingSource === 'HIE' ? 'hie' : 'standard'
+        }.label`,
+        {
+          defaultValue:
+            'Include as much detail as is necessary to explain the activity.'
+        }
+      ),
+    [activity.fundingSource]
+  );
+  const descriptionHint = useMemo(
+    () =>
+      t(
+        `activities.overview.activityDescriptionInput.${
+          activity.fundingSource === 'HIE' ? 'hie' : 'standard'
+        }.hint`,
+        {
+          defaultValue: ''
+        }
+      ),
+    [activity.fundingSource]
+  );
+  const syncDescription = useCallback(
+    html => {
+      updateActivity(activity.key, { description: html });
+    },
+    [activity.key]
+  );
+
+  const alternativesLabel = useMemo(
+    () =>
+      t('activities.overview.activityAlternativesInput.label', {
+        defaultValue: 'Provide a short overview of the activity.'
+      }),
+    []
+  );
+  const alternativesHint = useMemo(
+    () =>
+      t('activities.overview.activityAlternativesInput.hint', {
+        defaultValue: ''
+      }),
+    []
+  );
+  const syncAlternatives = useCallback(
+    html => {
+      updateActivity(activity.key, { alternatives: html });
+    },
+    [activity.key]
+  );
+
+  return (
+    <Subsection resource="activities.overview" nested>
+      <TextField
+        label={overviewLabel}
+        hint={overviewHint}
+        multiline
+        max={280}
+        rows={6}
+        className="data-entry-box"
+        value={summary}
+        onChange={overviewOnChange}
+      />
+
+      <div className="data-entry-box">
+        <FormLabel hint={descriptionHint}>{descriptionLabel}</FormLabel>
+        {activity.fundingSource === 'HIE' && (
+          <Instruction source="activities.overview.activityDescriptionInput.hie" />
```

Comment:
HIE activities have additional helptext that the `FormLabel` doesn't support.  If we have that pattern a lot, we should consider wrapping `FormLabel`, but this felt fine for a one-off.